configurations/es5.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+    extends: [
+        '../rules/eslint/best-practices/on.js',
+        '../rules/eslint/possible-errors/on.js',
+        '../rules/eslint/ecma-script-6/off.js',
+        '../rules/eslint/node-js-and-common-js/off.js',
```

Comment:
Yes because the idea is to always ship a full set of rules including on and off. 

Like this we are explicitly ignoring `ecma-script-6` and `node-js-and-common-js` rules.